lib/cli/parse.js

```diff
@@ -39,6 +37,12 @@ const parser = require('yargs')('run --help')
       type: 'array',
     },
   })
+  .coerce('config', configPath => {
```

Comment:
small nit - could we move this new logic to where the `options.c.transform` function previously was to keep all the option configurations grouped together?

Reply:
@camacho This is deprecated by `lib/cli/config.js` and `cosmiconfig`.